velox/docs/configs.rst

```diff
@@ -364,7 +368,10 @@ Hive Connector
      - integer
      - 128MB
      - Maximum distance in bytes between chunks to be fetched that may be coalesced into a single request.
-
+   * - file_writer_flush_threshold_bytes
```

Comment:
Does this config apply only when writer_spill_enabled is true? If so, let's clarify that.